integration_test/open_group_test.js

```diff
@@ -50,64 +50,64 @@ describe('Open groups', function() {
 
     // validate open chat has been added
     await app.client.waitForExist(
-      ConversationPage.rowOpenGroupConversationName(common.VALID_GROUP_NAME),
+      ConversationPage.rowOpenGroupConversationName(name),
       4000
     );
+  }
 
-    await common.timeout(1000);
+  it('openGroup: works with valid open group url', async () => {
+    await joinOpenGroup(common.VALID_GROUP_URL, common.VALID_GROUP_NAME);
   });
 
-  it('cannot join two times the same open group', async () => {
-    await app.client.element(ConversationPage.globeButtonSection).click();
-    await app.client.element(ConversationPage.joinOpenGroupButton).click();
-
-    await app.client
-      .element(ConversationPage.openGroupInputUrl)
-      .setValue(common.VALID_GROUP_URL2);
-    await app.client.element(ConversationPage.joinOpenGroupButton).click();
-    // first add is a success
-    await common.timeout(2000);
-    await app.client.waitForExist(
-      ConversationPage.rowOpenGroupConversationName(common.VALID_GROUP_NAME2),
-      8000
-    );
+  it('openGroup: cannot join two times the same open group', async () => {
+    await joinOpenGroup(common.VALID_GROUP_URL2, common.VALID_GROUP_NAME2);
 
     // adding a second time the same open group
     await app.client.element(ConversationPage.globeButtonSection).click();
     await app.client.element(ConversationPage.joinOpenGroupButton).click();
 
-    await app.client
-      .element(ConversationPage.openGroupInputUrl)
-      .setValue(common.VALID_GROUP_URL2);
+    await common.setValueWrapper(
+      app,
+      ConversationPage.openGroupInputUrl,
+      common.VALID_GROUP_URL2
+    );
     await app.client.element(ConversationPage.joinOpenGroupButton).click();
     // validate session loader is not shown
     await app.client.isExisting(ConversationPage.sessionLoader).should
       .eventually.be.false;
 
     await app.client.waitForExist(
       ConversationPage.sessionToastJoinOpenGroupAlreadyExist,
-      1000
+      1 * 1000
     );
 
     // validate overlay is still opened
     await app.client.isExisting(ConversationPage.leftPaneOverlay).should
       .eventually.be.true;
   });
 
-  it('can send message to open group', async () => {
+  it('openGroup: can send message to open group', async () => {
     // join dev-chat group
     await app.client.element(ConversationPage.globeButtonSection).click();
     await app.client.element(ConversationPage.joinOpenGroupButton).click();
 
-    await app.client
-      .element(ConversationPage.openGroupInputUrl)
-      .setValue(common.VALID_GROUP_URL2);
+    await common.setValueWrapper(
+      app,
+      ConversationPage.openGroupInputUrl,
+      common.VALID_GROUP_URL2
+    );
     await app.client.element(ConversationPage.joinOpenGroupButton).click();
-    // first add is a success
-    await common.timeout(2000);
+
+    // wait for toast to appear
+    await app.client.waitForExist(
+      ConversationPage.sessionToastJoinOpenGroupSuccess,
+      30 * 1000
```

Comment:
same as above. current test timeout is 30s https://github.com/loki-project/session-desktop/pull/1044/files?file-filters%5B%5D=.js#diff-dd50b247dac58a9b47120e5c4f1b2c46R9